app/alerts.py

```diff
@@ -19,25 +19,28 @@
 # Various modules provided by Dash to build the page layout
 import dash_core_components as dcc
 import dash_html_components as html
+import dash_bootstrap_components as dbc
 import dash_leaflet as dl
 import dash_leaflet.express as dlx
 
 # From navbar.py to add the navigation bar at the top of the page
 from navbar import Navbar
 
 # Various imports from utils.py, useful for both Alerts and Risks dashboards
-from utils import map_style, build_info_object
+from utils import map_style, build_info_object, build_live_alerts_metadata
 
 
 # ------------------------------------------------------------------------------
+
 # Map layer
 # The following block is used to determine what layer we use for the map and enable the user to change it
 
 # This function creates the button that allows users to change the map layer (satellite or topographic)
 def build_layer_style_button():
 
     button = html.Button(children='Activer la vue satellite',
-                         id='layer_style_button')
+                         id='layer_style_button',
+                         className="btn btn-warning")
```

Comment:
c'est la première fois que l'on ajoute ce `className`, non ? quel est l'intérêt ?

Reply:
@nrslt thanks for reviewing, I'd rather that we keep on commenting in english for unity purposes and foreign contributions that may occur 👍

This className allow to custom the button color while using  html.Button as illustrated [here](https://getbootstrap.com/docs/4.0/components/buttons/)